Library/docs/source/manual/documentation/expected_application_flow.md

```diff
@@ -61,7 +61,7 @@ void SpaceSystem::EnterSpace(const String& SpaceId, NullResultCallback Callback)
 
 This call does several things:
 * First, it validates the user calling it has the relevant permissions to be in the space given. If not it will reject the call. 
-* Secondly, it sets ‘scopes’, which are a subscription to a set of Entity replication messages. At the time of writing there is a single scope for a space, so any Entity in that space is replicated to any other user in the space. This is due to change in future, with more complex scopes representing ‘layers’ that can be used to alter what data is replicated to which user.
+* Secondly, it sets ‘scopes', which are a subscription to a set of Entity replication messages. At the time of writing there is a single scope for a space, so any Entity in that space is replicated to any other user in the space. This is due to change in future, with more complex scopes representing ‘layers' that can be used to alter what data is replicated to which user.
```

Comment:
[question] you only changed the closing one, was this intended?

Reply:
Fixed in latest, thanks!